lib/broadside/configuration/deploy_config.rb

```diff
@@ -22,14 +22,18 @@ class DeployConfig < ConfigStruct
         :env_vars,
         :command,
         :instance,
-        :predeploy_commands
+        :predeploy_commands,
+        :service_config,
+        :task_definition_config
       )
 
       TARGET_ATTRIBUTE_VALIDATIONS = {
         scale: ->(target_attribute) { validate_types([Fixnum], target_attribute) },
         env_file: ->(target_attribute) { validate_types([String], target_attribute) },
         command: ->(target_attribute) { validate_types([Array, NilClass], target_attribute) },
-        predeploy_commands: ->(target_attribute) { validate_predeploy(target_attribute) }
+        predeploy_commands: ->(target_attribute) { validate_predeploy(target_attribute) },
+        service_config: ->(target_attribute) { validate_types([Hash, NilClass], target_attribute) },
```

Comment:
to be honest i don't really see much point to the `EcsConfig class` and sort of think all that stuff belongs in the targets.  

right now it would be impossible to have a single broadside config file that can deploy to two clusters, as well has having this random extra special config class instead of just a key/value in the targets... and for what?